packages/suite-desktop/src/support/DesktopUpdater.tsx

```diff
@@ -191,18 +193,13 @@ const DesktopUpdater = () => {
         window.desktopApi!.on('update/downloading', (progress: UpdateProgress) =>
             downloading(progress),
         );
+        window.desktopApi!.on('update/error', (err: Error) => error(err));
```

Comment:
unnecessary function wrapping, you can use it like:
`window.desktopApi!.on('update/error', error);`

in fact all `window.desktopApi` listeners could be reduced like this (i didnt notice that before)

Reply:
Indeed! Force of habit :smile: